plugin.video.supertennis/resources/language/English/strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
It would be nice do drop gotham and move the addon to higher versions of Kodi only. In the process change the layout of strings.xml to strings.po as it is seriously deprecated :)